charts/k8s-service/values.yaml

```diff
@@ -325,6 +325,8 @@ minPodsAvailable: 0
 #       targetPort: http
 #       protocol: TCP
 #   sessionAffinity: ClientIP
+#   externalTrafficPolicy: Cluster
```

Comment:
These should probably be documented in the `service` documentation section above with respective links for 
- `externalTrafficPolicy`: https://kubernetes.io/docs/concepts/services-networking/service/#external-traffic-policy
- `internalTrafficPolicy`: https://kubernetes.io/docs/concepts/services-networking/service/#internal-traffic-policy